### PR TITLE
Add to onboarding reproduction logs

### PR DESCRIPTION
**Pull request by ru5h16h**

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ We take the average of the scores across all queries (6980 in this case), and we
 You can find this run on the [MS MARCO Passage Ranking Leaderboard](https://microsoft.github.io/MSMARCO-Passage-Ranking-Submissions/leaderboard/) as the entry named "BM25 (Lucene8, tuned)", dated 2019/06/26.
 So you've just reproduced (part of) a leaderboard submission!
 
-We can also use the official TREC evaluation tool, `trec_eval`, to compute other metrics than MRR@10. 
+We can also use the official [TREC](https://en.wikipedia.org/wiki/Text_Retrieval_Conference) evaluation tool, `trec_eval`, to compute other metrics than MRR@10.
```
Comment (lintool) [inline docs/experiments-msmarco-passage.md:233]: good call on the link, but let's link to official webpage? https://trec.nist.gov/

**Review by lintool — CHANGES_REQUESTED** [review]
Can you update comments with system config, etc. Notes about any issues?

**Comment by lintool** [comment]
@ru5h16h CR submitted. Please address if you're going to re-open PR.

**Comment by ru5h16h** [comment]
There was actually a discrepancy related to the account in this PR. The commit was created from a different account - ru5h1bh (which might have been set globally on the system) as opposed to the account on which I forked the repo (ru5h16h). I intended to use the latter one. Will raise a new PR.

**Comment by lintool** [comment]
ack

File: docs/experiments-msmarco-passage.md (hunk 1 above)
Comment (ru5h16h) [reply docs/experiments-msmarco-passage.md:233]: Sure.

**Pull request closed without merging**